core/assets/bundles/bundle_fr.properties

```
@@ -1832,19 +1832,19 @@ hint.depositItems = Pour déposer des ressources dans le Noyau, déplacez-les de
 hint.respawn = Pour réapparaître en tant que vaisseau, pressez [accent][[V][].
 hint.respawn.mobile = Vous avez pris le contrôle d'une unité/structure. Pour réapparaître en tant qu'unité de base, [accent]touchez l'avatar en haut à gauche.[]
 hint.desktopPause = Pressez [accent][[Espace][] pour mettre en pause et reprendre le jeu.
-hint.breaking = Maintenez [accent]Clic-droit[] pour détruire des blocs.
+hint.breaking = Maintenez [accent][[Clic-droit][] pour détruire des blocs.
```

```suggestion
hint.breaking = Maintenez [accent]Clic-droit[] pour détruire des blocs.
```
Ce changement n'est pas dans le bundle original.
https://github.com/phinner/mindustry/blob/e4253bcbf8fdc638cc52247ac553b5ed697447d4/core/assets/bundles/bundle.properties#L1835